inc/posttype/namespace.php

```diff
@@ -38,131 +38,131 @@ function list_post_types() {
 function register_post_types() {
 	register_extended_post_type(
 		'chapter', [
-			'admin_cols' => [
+			'admin_cols'            => [
 				'chapter_type' => [
 					'taxonomy' => 'chapter-type',
 				],
 			],
-			'quick_edit' => false,
-			'capability_type' => 'post',
-			'has_archive' => true,
-			'hierarchical' => false,
-			'menu_icon' => 'dashicons-book',
-			'supports' => [ 'title', 'editor', 'author', 'comments', 'page-attributes', 'revisions' ],
-			'show_in_menu' => false,
-			'show_in_admin_bar' => true,
-			'show_in_rest' => true,
-			'rest_base' => 'chapters',
+			'quick_edit'            => false,
+			'capability_type'       => 'post',
+			'has_archive'           => true,
+			'hierarchical'          => false,
+			'menu_icon'             => 'dashicons-book',
+			'supports'              => [ 'title', 'editor', 'author', 'comments', 'page-attributes', 'revisions' ],
+			'show_in_menu'          => false,
+			'show_in_admin_bar'     => true,
+			'show_in_rest'          => true,
+			'rest_base'             => 'chapters',
```

Comment:
[Our coding standards don't enforce this spacing rule](https://github.com/pressbooks/coding-standards/blob/f09f0ba4dd3d20c2acb63c13b28a14bfe86ed229/Pressbooks/ruleset.xml#L27)?

Looking at this part of the diff, and this file in general, it looks like nothing has changed except the spaces for many lines?  I would prefer this kind of alignment be reverted so that the diff is more clear.

For devs, there was a composer vendor update that started enforcing these kinds of spaces, it wasn't the case before. and we made ignoring this rule more explicit.  If this is failing for our current CI tests I'd like to know?